docs/job-item-flow.md

```diff
@@ -0,0 +1,15 @@
+# Information flow of a Job Item
+
+To make the best use of Redis, we propose you to configure your application to work in the manner below.
+
+Once a Job is submitted, the workload is prepared in Redis, who should have three queues: *main*, *processing* e *results*. These queues represent the state of each item put in them.
```

Comment:
"who should have three queues"-> "which should have..";
"processing e results" -> "processing and results";
"item put in them" -> "item there";

Reply:
You are right! Fixed in commit 59cdc4d104b8a0e7ba3f89aab68eac8ee3d714b7